src/Hng.Application/Hng.Application.csproj

```diff
@@ -27,7 +27,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Features\UserManagement\Commands\" />
```

Comment:
Please do get latest from dev to prevent removing folder paths in the item group.

Reply:
well, i pulled from Dev before pushing.

Thanks tho